modules/mod.rb

```diff
@@ -74,6 +74,40 @@ module Mod
 				event.respond('❌ Invalid argument. Please mention a valid user.')
 			end
 		end
+	  
+		command(:warn, description: 'Warn somebody on the server. Mod only.', usage: "#{Config.prefix}warn @user reason", min_args: 2) do |event, *kick_reason|
+			unless Helper.is_helper?(event) || Helper.is_moderator?(event) || Helper.is_developer?(event) || Helper.is_admin?(event.user)
+				event.respond("âŒ You don't have permission for that!")
+				break
+			end
+
+			member = event.server.member(event.message.mentions[0])
+
+			break if event.channel.private?
+			if event.message.mentions[0]
+				final_message = kick_reason.drop(1)
+				display = final_message.join(' ')
+				message = "You have been warned on the server **#{event.server.name}** by #{event.message.author.mention} | **#{event.message.author.display_name}**\n"
+				message << "They gave the following reason: ``#{display}``"
+				begin
+						member.pm(message)
+				rescue Discordrb::Errors::NoPermission
+						event.respond('Could not DM user about warn reason!')
+						break
+				end
+				begin
+					# Register for logging
+					Logging.record_action('kick', event.user, member, display)
```

Comment:
Though keep this line. Might wanna change it from kick to warn too. I'm gonna have to revamp this system this weekend.